python/tests/test_frames.py

```diff
@@ -162,3 +166,46 @@ def test_wrong_linkage(assert_log):
 
     f.link([f])
     assert_log("wrong linkage")
+
```

Comment:
Thought:
Do you want to add pandas tests with index-no-index to have confirmation that intended functionality also works?

Reply:
Na, that is outside of the scope of dlisio. The current scope is to serve a numpy array with no assumption on how that will be used later on. I don't we should try to make any guarantees for anything that end-user do after we serve them the curve